core/storage/user/gae_models.py

```diff
@@ -1092,6 +1157,12 @@ def apply_deletion_policy(cls, user_id):
         ndb.delete_multi(
             cls.query(cls.user_id == user_id).fetch(keys_only=True))
 
+    @staticmethod
+    def get_export_policy():
+        """Model contains user data."""
+        return base_models.EXPORT_POLICY.CONTAINS_USER_DATA
+
```

Comment:
Drop newline between adjacent methods. Please fix here and elsewhere (there are multiple occurrences).

Reply:
Done.